lib/tcp.js

```diff
@@ -157,7 +157,10 @@ exports.client = function (options, transportUtil) {
       reconnect.on('disconnect', function (err) {
         seneca.log.debug('client', type, 'disconnect', spec, topic, clientOptions,
            (err && err.stack) || err)
-        _.find(connections, { clientOptions: clientOptions }).setup = false
+        var connections = _.find(connections, { clientOptions: clientOptions });
```

Comment:
@friedemannsommer i followed the example on https://github.com/senecajs-attic/getting-started/blob/master/app.js

`var SenecaWeb = require('seneca-web')
var Express = require('express')
var Router = Express.Router
var context = new Router()

var senecaWebConfig = {
      context: context,
      adapter: require('seneca-web-adapter-express'),
      options: { parseBody: false } // so we can use body-parser
}

var app = Express()
      .use( require('body-parser').json() )
      .use( context )
      .listen(3000)

var seneca = require( 'seneca' )()
      .use( SenecaWeb, senecaWebConfig )
      .use( 'api' )
      .client( { type:'tcp', pin:'role:math' } )`